docs/specification/1.0.md

```diff
@@ -464,6 +464,23 @@ Field | Optionality | Type | Description
 <nobr>`label`</nobr>| REQUIRED | *string* | A short, human-readable label to display for the source of the information displayed on this card. If a `url` is also specified, this MAY be the text for the hyperlink.
 `url` | OPTIONAL | *URL* | An optional absolute URL to load (via `GET`, in a browser context) when a user clicks on this link to learn more about the organization or data set that provided the information on this card. Note that this URL should not be used to supply a context-specific "drill-down" view of the information on this card. For that, use `link.url` instead.
 `icon` | OPTIONAL | *URL* | An absolute URL to an icon for the source of this card. The icon returned by this URL SHOULD be a 100x100 pixel PNG image without any transparent regions.
+`topic` | OPTIONAL | *[FHIR Coding](http://hl7.org/fhir/datatypes.html#coding)* | A "FHIR Coding" json object containing a descriptive topic related to the content of the card. A topic provides a high-level categorization that can be useful for filtering and searching.
```

Comment:
Categorization could clearly be helpful.  Should we say _something_ in here about where these Codings come from, even if it's around mutual agreement between clients/servers.  There will be questions around this field, for sure, but having no comments about coding systems or where this comes from makes it more ambiguous.

Reply:
Moved conversation to zulip.